app/assets/javascripts/app/form-validation.js

```diff
@@ -21,7 +20,6 @@ const validate = {
   },
 
   addEvents() {
-    this.form.addEventListener('change', e => validateField(e.target));
```

Comment:
Do you want to submit a test to this branch? This line of code is what was causing the double submit bug, but if I have a test for it then I can refactor so the bug no longer occurs